osbuild/monitor.py

```diff
@@ -317,7 +317,7 @@ def __init__(self, fd: int, manifest: osbuild.Manifest):
     def begin(self, pipeline: osbuild.Pipeline):
         self._context.set_pipeline(pipeline)
         if pipeline.stages:
-            self._progress.sub_progress = Progress("stages", len(pipeline.stages))
+            self._progress.sub_progress = Progress(f"pipeline: {pipeline.name}", len(pipeline.stages))
```

Comment:
Thank you for this change! This looks nice and useful but slightly unrelated to this PR, would you mind opening it as it's own PR and remove from this one? I think this can go in quickly :)

Reply:
Sure - the "progress forwarding PR" will be separate, thanks for finding this mixup!